lib/storage/client.js

```diff
@@ -1044,6 +1044,32 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * createGroup is a static method that creates a groups and sends it to TozStore,
+   * allowing you to supply your own encryptionKey pair.
+   *
+   *
+   * @param {object} data  A hashmap of the data in the group - for now this just includes groupName
+   * @param {string} recipientEncryptionKey The public encryption key of the reader of this note
+   *
+   * @returns {Group} A response from TozStore; the group that has been written.
+   */
+  async createGroup(data, recipientEncryptionKey) {
```

Comment:
what is the `recipientEncryptionKey` here? Ganearlly in SDK method's we'll refer to clients by client ID, but in createGroup I would think the client would add themselves as the 'recipient' in which case we could omit this parameter, right?